frontend/tests/e2e_tests/integration/99-settings.spec.ts

```diff
@@ -183,8 +185,9 @@ test.describe('Settings', () => {
       await page.getByText(/Enable Two Factor/).click();
       await page.waitForTimeout(timeouts.default);
     });
-    test('allows logging in without 2fa after deactivation', async ({ baseUrl, context, environment, password, username }) => {
+    test('allows logging in without 2fa after deactivation', async ({ baseUrl, browser, environment, password, username }) => {
       test.skip(environment !== 'staging');
+      let context = await browser.newContext({ storageState: { ...emptyStorageState } });
```

Comment:
directly created context should be closed according to the docs
https://playwright.dev/docs/api/class-browser#browser-new-context

Reply:
done, but unfortunately there's not too much for us to gain there, as the contexts are usually created to work around the storage state (which then again interferes with the test recordings etc. 😞)